### DISCUSSION TITLE
Drag and Drop In the creator kit

### DISCUSSION THREAD

**Discussion by Jswanson-Dev** [microsoft/powercat-creator-kit#126] [General]
I would like to see Scotts Drag and Drop control in the kit. That is one of the features I get asked about the most.

**Comment by denise-msft** [comment]
Can you please upvote this [existing Feature request for drag and drop](https://github.com/microsoft/powercat-creator-kit/issues/108) if the description there makes sense for you?

For feature requests in general, submit them through the [Feature request Issues template](https://aka.ms/CreatorKit/FeatureRequest) (not discussion, please). 

We really appreciate your feedback!
